src/main/java/mate/academy/controller/BookController.java

```diff
@@ -25,36 +27,48 @@
 @Tag(name = "Book management", description = "Endpoints for managing books")
 @RequiredArgsConstructor
 @RestController
-@RequestMapping("/books")
+@RequestMapping("/api/books")
```

Comment:
```suggestion
@RequestMapping("/books")
```

Add `server.servlet.context-path=/api` to application.properties